src/SDKs/Azure.Core/data-plane/Azure.Core/Pipeline/HttpPipelineOptions.cs

```diff
@@ -4,21 +4,25 @@
 using System;
 using System.Collections.Generic;
 using System.ComponentModel;
+using Azure.Core.Pipeline.Policies;
 
 namespace Azure.Core.Pipeline
 {
     public class HttpClientOptions
     {
         private HttpPipelineTransport _transport = HttpClientTransport.Shared;
 
+        public HttpClientOptions()
+        {
```

Comment:
It would be nice to see a test case that confirms telemetry can be disabled, even if that is just by setting to `null`.   If nothing else, it would help to provide context and serve as an example.